adapters/tuya/__init__.py

```diff
@@ -3,6 +3,7 @@
 from adapters.tuya.TS0013 import TS0013
 from adapters.tuya.TS0015 import TS0015
 from adapters.tuya.TS0601 import TS0601
+from adapters.tuya.TV02 import TV02
```

Comment:
I don't see this file in the PR, could you please double-check that all required changes are included